docs/resources/monitor_alert.md

```diff
@@ -4,9 +4,12 @@ page_title: "DigitalOcean: digitalocean_monitor_alert"
 
 # digitalocean_monitor_alert
 
-Provides a [DigitalOcean Monitoring](https://docs.digitalocean.com/reference/api/api-reference/#tag/Monitoring) resource.
-Monitor alerts can be configured to alert about, e.g., disk or memory usage exceeding certain threshold, or traffic at certain
-limits. Notifications can be sent to either an email address or a Slack channel.
+Provides a [DigitalOcean Monitoring](https://docs.digitalocean.com/reference/api/api-reference/#tag/Monitoring)
+resource. Monitor alerts can be configured to alert about, e.g., disk or memory
+usage exceeding certain threshold, or traffic at certain limits. Notifications
```

Comment:
Potential typo, should it be something like
```suggestion
usage exceeding a certain threshold, or traffic at certain limits. Notifications
```
or
```suggestion
usage exceeding certain thresholds, or traffic at certain limits. Notifications
```
?

Reply:
:+1:  Good catch. Just wrapped that line without really reading it. Pushed a change.